compass/landice/tests/calving_dt_convergence/__init__.py

```diff
@@ -0,0 +1,38 @@
+from compass.testgroup import TestGroup
+from compass.landice.tests.calving_dt_convergence.dt_convergence_test \
+     import DtConvergenceTest
+
+
+class CalvingDtConvergence(TestGroup):
+    """
+    A test group for MISMIP+ test cases.
+    This test group uses a pre-made mesh file.
+    """
+    def __init__(self, mpas_core):
+        """
+        mpas_core : compass.landice.Landice
+            the MPAS core that this test group belongs to
+        """
+        super().__init__(mpas_core=mpas_core, name='calving_dt_convergence')
+
+        for mesh in [
+                     'mismip+',
+                     'humboldt',
+                     'thwaites'
+                    ]:
+            for calving in [
+                            'specified_calving_velocity',
+                            'von_Mises_stress',
+                            'eigencalving'
+                           ]:
+                for velo in [
+                             'none',
+                             'FO'
+                            ]:
+                    if (calving == 'specified_calving_velocity' and
+                       velo == 'FO'):
+                        continue  # This combination is not useful
```

Comment:
Why is this not useful?

Reply:
I was just trying to minimize adding too many new test cases, especially the FO ones, which are quite slow to run.  We can add them if you think it's worthwhile.

Reply:
No, I convinced myself that it's fine as-is, since we care about calving velocity and not retreat velocity for this CFL condition.